index.html

```diff
@@ -99,17 +99,33 @@ <h3>Extensions to the <code>Window</code> interface</h3>
           <dl>
             <dt><dfn data-dfn-for="Window">visualViewport</dfn></dt>
             <dd>
-              If the <code>window</code> has a <dfn data-cite="!HTML#browsing-context">Browsing Context</dfn> set,
-              return the <a>VisualViewport</a> object associated with it. Otherwise, return null.
+              If the <a>window</a>'s associated <dfn data-cite="!HTML#document">Document</dfn> is
```

Comment:
"associated Document", here and elsewhere, should link to https://html.spec.whatwg.org/#concept-document-window

Reply:
Done.